styles.css

```diff
@@ -76,7 +77,7 @@ footer{
 @media (max-width: 600px) {
 
     header{
-        background-color: aquamarine;
+        background-color: #ac4449;
     }
 
     body{
```

Comment:
grid template had 
a
b
c
d
.
f

mogen zijn, die 3x is niet nodig het betreft maar 1 fr